pygfx/materials/_text.py

```diff
@@ -0,0 +1,50 @@
+from ._base import Material
+from ..utils import unpack_bitfield, Color
+
+
+class TextMaterial(Material):
+    """The default material used by Text."""
+
+    uniform_type = dict(
+        color="4xf4",
+    )
+
+    def __init__(self, color=(1, 1, 1, 1), screen_space=True, **kwargs):
+        super().__init__(**kwargs)
+
+        self._screen_space = None
+        self.screen_space = screen_space
+        self.color = color
+
+    def _wgpu_get_pick_info(self, pick_value):
+        # This should match with the shader
+        _ = unpack_bitfield(pick_value, wobject_id=20, index=26, x=9, y=9)
+        return {}
+
+    @property
+    def screen_space(self):
+        """Whether the text is applied in screen space (in contrast to model space)."""
```

Comment:
Is this equivalent to using an ndc camera?

Reply:
It means that the text is flat on the screen, no matter the orientation of the text-object and camera. Similar to how Points are represented with a circle in screen space. This is good for labeling stuff.

If `False`, the text behaves more like other objects, so you can e.g. put it on a surface in a 3D scene.

I had not previously realized that we'd have both use-cases. I think I like the idea of creating separate materials for these, because their use is profoundly different.

Reply:
Ah I see! Very useful:)

Reply:
Thinking of the easiest way to communicate this in docstrings, I'm reminded of the graphics term "billboard" which is used to describe objects that are always rendered facing the camera.